src/templates/tabs/coldwallet.jade

```diff
@@ -78,5 +78,7 @@ container(ng-controller="ColdWalletCtrl")
                 td {{line.specification.authorized}}
                 td {{line.warning}}
     div.row.row-content
-      div.col-xs-12
-        button.btn.btn-success(ng-click='gotoLogin()') Back to login
\ No newline at end of file
+      div.col-xs-12.col-sm-2
+        button.btn.btn-success(ng-click='gotoLogin()') Back to login
+      div.col-xs-12.col-sm-2
+        button.btn.btn-success(ng-click='refresh()') Refresh
```

Comment:
Just a stylistic thing, but I think it would be better if we have the refresh button first and then the back to login button after.